src/models/ICard.ts

```diff
@@ -19,3 +19,11 @@ export interface ICard {
   discount: number;
   id: string;
 }
+
+export interface IDropDown {
+  tag: string;
+  menu: Array<{
```

Comment:
make a separate interface for the menu. You will have  
IDropDown {
tag: string;
many: IDropDownMenu[]
}